wandb/sdk/launch/runner/sagemaker_runner.py

```diff
@@ -74,32 +66,49 @@ def get_status(self) -> Status:
         return self._status
 
 
-class AWSSagemakerRunner(AbstractRunner):
+class SagemakerRunner(AbstractRunner):
```

Comment:
SageMaker?

Reply:
oh yeah I guess that is how they do the capitalization on their site https://aws.amazon.com/sagemaker/